include/vapor/ParamsMgr.h

```diff
@@ -523,6 +527,10 @@ class PARAMS_API ParamsMgr : public MyBase {
  void RegisterStateChangeCB(std::function<void()> callback) {
 	_ssave.RegisterStateChangeCB(callback);
  }
+    
+    void RegisterIntermediateStateChangeCB(std::function<void()> callback) {
```

Comment:
The documentation for the function right above, RegisterStateChangeCB, just says that it registers a callback that will be called when the state changes. That can be inferred from the function name although I think it could be improved by renaming it to RegisterIntermediateStateChangeCallbackFunction. You could argue that you should specify what an intermediate change is. However that has nothing to do with the job of this function. That should be described in the class description so I was going to do that but it turns out all of the documentation is split up amongst the functions so you have to read the documentation for each of the functions to figure out what the class does so I copied it from the IntermediateChange function.